WordPress/src/main/java/org/wordpress/android/ui/posts/PostsListFragment.java

```diff
@@ -609,6 +664,21 @@ public void onPostChanged(OnPostChanged event) {
                 hideLoadMoreProgress();
                 if (!event.isError()) {
                     mCanLoadMorePosts = event.canLoadMore;
+                    int count;
```

Comment:
I think we can safely do without these changes in `onPostChanged`. They should have been removed when `onPostsLoaded` was brought back.